content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,205 @@
+For example if you followed the instructions to 
+[install containerd](https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd) 
+replace the `containerd-shim-runhcs-v1.exe` is installed at `$Env:ProgramFiles\containerd` with the newly built shim.
+
+We are working to improve this process by enabling HostProcess support directly in 
+[containerd via the CRI api](https://github.com/containerd/containerd/pull/5131) 
+as well as shipping a version of the containerd shim for Windows with support direclty via containerd releases.
+
+## Creating a HostProcess Pod
+
+### Security Policy Config
+Enabling a Windows HostProcess pod simply requires setting the right configurations in the pod security configuration. 
+Many of the policies in the [Pod Security Standards](/docs/concepts/security/pod-security-standards) do not apply to 
+HostProcess containers (and Windows in general) due to architectural differences between Windows and Linux. As such, 
+here are the set of policies and configuration values required to enable HostProcess pods:
+
+<table>
+	<caption style="display:none">Privileged policy specification</caption>
+	<tbody>
+		<tr>
+			<td><strong>Control</strong></td>
+			<td><strong>Policy</strong></td>
+		</tr>
+		<tr>
+			<td style="white-space: nowrap"><a href="/docs/concepts/security/pod-security-standards">Windows HostProcess</a></td>
+			<td>
+				<p>Windows pods offer the ability to run <a href="/docs/tasks/configure-pod-container/create-hostprocess-pod">
+        HostProcess containers</a> which enables privileged access to the Windows node. </p>
+				<p><strong>Allowed Values</strong></p>
+				<ul>
+					<li><code>true</code></li>
+				</ul>
+			</td>
+		</tr>
+		<tr>
+			<td style="white-space: nowrap"><a href="/docs/concepts/security/pod-security-standards">Host Networking</a></td>
+			<td>
+				<p>Will be in host network by default initially. Support 
+				to set network to a different compartment may be desirable in 
+				the future.</p>
+				<p><strong>Allowed Values</strong></p>
+				<ul>
+					<li><code>true</code></li>
+				</ul>
+			</td>
+		</tr>
+    <tr>
+			<td style="white-space: nowrap"><a href="/docs/tasks/configure-pod-container/configure-runasusername/">runAsUsername</a></td>
+			<td>
+				<p>Specification of which user the HostProcess container should run as is required for the pod spec.</p>
+				<p><strong>Allowed Values</strong></p>
+				<ul>
+          <li><code>"NT AUTHORITY\\SYSTEM"</code></li>
+					<li><code>"NT AUTHORITY\\Local service"</code></li>
+          <li><code>"NT AUTHORITY\\NetworkService"</code></li>
+				</ul>
+			</td>
+		</tr>
+    <tr>
+			<td style="white-space: nowrap"><a href="/docs/concepts/security/pod-security-standards">runAsNonRoot</a></td>
+			<td>
+				<p>HostProcess containers have privileged access to the host so the runAsNonRoot field cannot be set to true.</p>
```

Comment:
```suggestion
				<p>Because HostProcess containers have privileged access to the host, the <tt>runAsNonRoot</tt> field cannot be set to true.</p>
```